src/server.cpp

```diff
@@ -2575,6 +2592,24 @@ void Server::DenySudoAccess(u16 peer_id)
 	Send(&pkt);
 }
 
+
+void Server::DenyAccessCompliant(u16 peer_id, u16 proto_ver, AccessDeniedCode reason,
```

Comment:
Ahh, I see... I thought it was a typo.
In that case maybe it should be DenyAccessVersionCompliant (so people know what's being compliant here)?